pynq/pl.py

```diff
@@ -536,7 +536,11 @@ def client_request(cls, address=PL_SERVER_FILE,
         None
 
         """
-        cls._remote = Client(address, family='AF_UNIX', authkey=key)
+        try:
+            cls._remote = Client(address, family='AF_UNIX', authkey=key)
+        except FileNotFoundError:
+            raise ConnectionError(
+                       "Could not connect to Pynq PL server") from None
```

Comment:
My worry with being more general in the except block is that there may be other circumstances where a different error may be thrown which may be caused by things other than PL server not being started. For example some form of serialiasation error caused by the PL server and the process running different versions of python. In these instances I think we would want to pass the full exception back up the stack. The aim for this commit is to very specifically change a FileNotFoundError on a file that the user has never seen and present something more useful.